focus-group.js

```diff
@@ -23,11 +27,18 @@ function findGroupNodeByEventTarget(target) {
     let node = target.closest(`[role=${role}]`)
     if (node) return node
   }
+
+  return null
 }
 
 function getItems(target, group) {
-  if (group.role === 'toolbar') return getToolbarItems(group)
-  return group.querySelectorAll(`[role=${target.role}]`)
+  if (group.hasAttribute('focusgroup')) {
+    return [...group.children]
```

Comment:
Are you sure, that all `children` should be focusable?

1. What is some `children` is not focusable (like `<h2>`)?
2. Can you double-check that we don't need similar logic to `getToolbarItems`?